### PR TITLE
Add hammer

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes Ready for review

File: recipes/hammer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    - pkg-config
+  host:
+    - python
+    - root_base
+    - root-binaries
+    - root-dependencies
+    - boost-cpp
+    - hepmc2
+    - yaml-cpp
+    - numpy =1.14
```
Comment (beckermr) [inline recipes/hammer/meta.yaml]: You should remove this pin. It will stomp on the global numpy pinnings.

Comment (chrisburr) [reply recipes/hammer/meta.yaml]: Done :+1:

The build might fail now as I think staged-recipes is inconsistent, or at least it tried to use 1.11 when I built this locally

Comment (beckermr) [reply recipes/hammer/meta.yaml]: Right ok. the global pinnings are higher.

**Comment by chrisburr** [comment]
Thanks @beckermr! 🍰

**Pull request merged**